### ISSUE TITLE
Is GPU needed?

### ISSUE THREAD

**Issue by ghost** [keunwoochoi/LSTMetallica#6]
Hi,

Do we need GPU to train and generate tracks using this code? I suppose without a GPU it would not be possible in a reasonable time to train the network.

Thanks.

**Comment by keunwoochoi** [comment]
Yes, as you think. Also because this code is not perfect and you would like to modify, which requires many runs of experiments.
